packages/11ty/_plugins/globalData/index.js

```diff
@@ -52,6 +55,31 @@ const validatePublication = (publication) => {
   return publication
 }
 
+/**
+ * @function validateConfig - throws error if data does not parse to config 
+ * @throws {Error}
+ * 
+ * @param {Object} config - Deserialized config data
+ * 
+ */
+const validateConfig = (config) => {
+
+  for (let [key,value] of Object.entries(config)) {
+    switch (key) {
+    case 'pdf':
+
+      // For now just use some quickie type-checking asserts
+      assert.strictEqual(typeof value.outputDir,'string',new TypeError("pdf.outputDir must be a string"))
+      assert.strictEqual(typeof value.filename,'string',new TypeError("pdf.filename must be a string"))
+      // FIXME: if pagePDF exists... assert.strictEqual(typeof value.pagePDF,'object',new TypeError("pdf.pagePdf must be an object"))
+      // FIXME: assetLinks should be an array of objects if it exists
+      break
```

Comment:
This might be a simple conditional block unless there are additional cases that we need to address? Suggested formatting/indentation:
```suggestion
      case 'pdf':
        assert.strictEqual(typeof value.outputDir, 'string', new TypeError('pdf.outputDir must be a string'))
        assert.strictEqual(typeof value.filename, 'string', new TypeError('pdf.filename must be a string'))
        // @fix if pagePDF exists... 
        // assert.strictEqual(typeof value.pagePDF,'object',new TypeError("pdf.pagePdf must be an object"))
        // assetLinks should be an array of objects if it exists
        break
```

Reply:
@mphstudios Makes sense -- done. I initially put it there thinking we might validate other things but the config doc is a bit too broad to treat that way (ie, `pdf` is the only config.yaml block with this much nested structure)